Exiled.API/Features/Npc.cs

```diff
@@ -49,7 +49,7 @@ public Npc(GameObject gameObject)
         /// <summary>
         /// Gets a list of Npcs.
         /// </summary>
-        public new List<Npc> List
+        public static new List<Npc> List
```

Comment:
That a Breaking change 

So we should just // TODO make this Static

Reply:
We can't access it in the patch without it being static, unless we create a new NPC just for doing the count subtraction, then destroy it after (which would be expensive for no good reason)